lib_test/test_parser_async.ml

```diff
@@ -0,0 +1,81 @@
+(*
+ * Copyright (c) 2012 Anil Madhavapeddy <anil@recoil.org>
```

Comment:
If you copy a file with that license and keep any code from the original, then you should keep the existing names.  When you add or modify code reasonably substantively, then append your name to the copyright list.  So in this case, it's probably just your name and the current year.